### DISCUSSION TITLE
Commercial license

### DISCUSSION THREAD

**Discussion by akssieg** [NVlabs/instant-ngp#933] [General]
The NVIDIA license mentions that it is not for commercial use. I want to know whether I can use this repo in my backend server and generate .mp4 videos and 3d model for my customers? Or is there any purchase fee?

**Comment by Tom94** [comment]
Hi, if you have customers you should ask for a commercial license using the form linked in the readme: https://www.nvidia.com/en-us/research/inquiries/

**Reply by akssieg** [reply]
Thanks
